SwiftMonkeyPaws/MonkeyPaws.swift

```diff
@@ -35,10 +31,60 @@ private let circleRadius: CGFloat = 7
     }
     ```
 */
+
 public class MonkeyPaws: NSObject, CALayerDelegate {
+
+    public struct Configuration {
+        // Customise the appearance of the paws
+        public struct Paws {
+
+            /// Define the colour of the Paws
+            ///
+            /// - randomized: random colour for each paw
+            /// - constant: same colour for the paws
+            enum Colour {
+                case randomized
+                case constant(UIColor)
+            }
+            // Colour of the paws
+            let colour: Colour
+
+            // Brightness of a particular paw
+            let brightness: CGFloat
+
+            // Maximum visible paws at one time
+            let maxShown: Int
+
+            public static let `default`: Paws = Paws(colour: .randomized,
```

Comment:
If you add the `public init()` for `Paws` you might as well provide default parameter values, like that:
```swift
public init(colour: Colour = .randomized, brightness: CGFloat = 0.5, maxShown: Int = 15) {
    self.colour = colour
    self.brightness = brightness
    self.maxShown = maxShown
}
```
The result is that writing `Paws.default` is equal to `Paws()` but with default initializer parameter values I think the usage is slightly more convenient. What do you think?